chef_master/source/custom_resources.rst

```diff
@@ -857,7 +980,7 @@ For example, the following properties define ``username`` and ``password`` prope
 .. end_tag
 
 Block Arguments
-+++++++++++++++++++++++++++++++++++++++++++++++++++++
+-----------------------------------------------------
```

Comment:
It seemed to me that Block Arguments described something about properties that was useful in load_current_value processing and so should have the same header as property_is_set? and reset_property.  Grouping Block Arguments under the property section was confusing.  I'm willing to change this.